content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

```diff
@@ -0,0 +1,133 @@
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>The DTM application is integrated with Gainsight to capture the data.</td>
+<td>CTM-130</td>
+</tr>
+
+<tr>
+<td>Added a warning notification if a custom property associated with an asset is deleted using Cumulocity REST APIs when an asset is viewed.</td>
+<td>CTM-141</td>
+</tr>
+
+<tr>
+<td>A confirmation pop-up is shown in the <b>Property library</b> page for every delete operation.</td>
+<td>CTM-159</td>
+</tr>
+
+<tr>
+<td>Implemented a maximum file size field for custom properties of type file.</td>
+<td>CTM-175</td>
+</tr>
+
+<tr>
+<td>Added a confirmation popup which states that the changes made to the asset type will not be reflected in the assets that are currently associated with it.</td>
+<td>CTM-144</td>
+</tr>
+
+<tr>
+<td>Added a confirmation popup which states that the changes made to the custom property will not be reflected in the assets that are currently associated with it's asset type.</td>
+<td>CTM-147</td>
+</tr>
+
+<tr>
+<td>An alert save pop-up is displayed in the localization page.</td>
+<td>CTM-155</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes in Digital Twin Manager
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>The <b>Import assets</b> button is not visible for the last asset in the hierarchy now.</td>
+<td>CTM-170</td>
+</tr>
+
+<tr>
+<td>The loading message in the localization page is corrected.</td>
+<td>CTM-154</td>
+</tr>
+
+<tr>
+<td>An issue observed during hierarchy creation is fixed.</td>
+<td>CTM-125</td>
+</tr>
+
+<tr>
+<td>An issue while creating subassets when there are both required and optional child asset types at same level is fixed.</td>
+<td>CTM-167</td>
+</tr>
+
+<tr>
+<td>Changes made in the asset type page are getting saved now.</td>
+<td>CTM-157</td>
+</tr>
+
+<tr>
+<td>Provided a fix for the Add child asset window which became unresponsive for assets that had a custom property deleted from the backend.</td>
+<td>CTM-196</td>
+</tr>
+
+<tr>
+<td>A warning message is now displayed on deletion of a custom property from the backend.</td>
+<td>CTM-200</td>
+</tr>
+
+<tr>
+<td>The properties tab is not displayed when there are no properties defined.</td>
+<td>CTM-174</td>
+</tr>
+
+<tr>
+<td>The text of the unassign device alert popup has been corrected.</td>
+<td>CTM-189</td>
+</tr>
+
+<tr>
+<td>Users are now able to delete the uploaded file in custom properties of an asset.</td>
+<td>CTM-198</td>
+</tr>
+
+<tr>
+<td>Fixed a bug where a bulk import failed if the complex property contains a duplicate label.</td>
+<td>CTM-195</td>
+</tr>
+
+<tr>
+<td>Fixed localization issues related to the bulk import feature.</td>
+<td>CTM-153</td>
+</tr>
+
+<tr>
+<td>The creation of assets via bulk import event is now getting triggered in the Gainsight application.</td>
```

Comment:
Again, I don´t think that Gainsight is a term familiar to the user.

Reply:
Explained about it